src/components/Form.js

```diff
@@ -0,0 +1,294 @@
+                className="form-control"
+                name="state"
+                placeholder="State"
+                required
+              />
+              <div className="invalid-feedback">
+                Please provide a valid state.
+              </div>
+              <div className="valid-feedback">Looks good!</div>
+            </Col>
+            <Col md="4" className="mb-3">
+              <label
+                htmlFor="defaultFormRegisterPasswordEx4"
+                className="grey-text"
+              >
+                Zip
+              </label>
+              <input
+                value={this.state.zip}
+                onChange={this.changeHandler}
+                type="text"
+                id="defaultFormRegisterPasswordEx4"
+                className="form-control"
+                name="zip"
+                placeholder="Zip"
+                required
+              />
+              <div className="invalid-feedback">
+                Please provide a valid zip.
+              </div>
+              <div className="valid-feedback">Looks good!</div>
+            </Col>
+          </Row>
+          <Row>
+          <Col md="4" className="mb-3">
+              <label
+                htmlFor="defaultFormRegisterConfirmEx3"
+                className="grey-text"
+              >
+                  Phone Number
+                  </label>
+                  <input
+                value={this.state.phonenumber}
+                onChange={this.changeHandler}
+                type="text"
+                id="defaultFormRegisterPasswordEx4"
+                className="form-control"
+                name="phonenumber"
+                placeholder="Phone Number"
+                required
+              />
+              <div className="invalid-feedback">
+                Phone Number
+              </div>
+              </Col>
+    <Col md="8"
+    className="mb-3">
+    <label
+      htmlFor="defaultFormRegisterConfirmEx3"
+      className="grey-text"
+    >
+        Message
+        </label>
+        <textarea
+      value={this.state.message}
+      onChange={this.changeHandler}
+      type="text"
+      id="defaultFormRegisterPasswordEx4"
+      className="form-control"
+      name="message"
+      placeholder="Enter Text"
+      required
+    />
+    </Col>
+  </Row>
+          <Col md="4" className="mb-3">
+            <div className="custom-control custom-checkbox pl-3">
+              <input
+                className="custom-control-input"
+                type="checkbox"
+                value=""
+                id="invalidCheck"
+                required
+              />
+              <label className="custom-control-label" htmlFor="invalidCheck">
+                Agree to terms and conditions
+              </label>
+              <div className="invalid-feedback">
+                You must agree before submitting.
+              </div>
+            </div>
```

Comment:
A link to the T&Cs needs to be added or the T&Cs need to be included next to the checkbox.